docs/gwcs/developer_introduction.rst

```diff
@@ -0,0 +1,195 @@
+.. _dev_intro:
+
+Introduction to Constructing Generalized World Coordinate System Models
+=======================================================================
+
+Pixel Conventions and Definitions
+---------------------------------
+
+:ref:`pixel-conventions-and-definitions`
+
+Installation
+------------
+
+:ref:`installing-gwcs`
+
+
+.. _getting-started:
+
+Basic Structure of a GWCS Object
+--------------------------------
+
+The key concept to be aware of is that a GWCS Object consists of a sequence
+of steps; each step contains a transform (i.e., an Astropy model) that
+converts the input coordinates of the step to the output coordinates of
+the step. Furthermore, each step has a required coordinate frame associated
+with the step. The coordinate frame represents the input coordinate frame, not
+the output coordinates. Most typically, the first step coordinate frame is
+the detector pixel coordinates (the default). Since no step has a coordinate
+frame for the output coordinates, it is necessary to append a step with no
+transform to the end of the pipeline to represent the output coordinate frame.
+For imaging, this frame typically references one of the Astropy standard
+Sky Coordinate Frames of Reference. The GWCS frames also serves as an 
+information containe, holding the units on the axes, the names of the axes, 
```

Comment:
```suggestion
information container, holding the units on the axes, the names of the axes, 
```

Reply:
As for pre-commit, I'll do that after this PR. More work is coming on the documentation so it can be dealt with in that.